.pullapprove.yml

```diff
@@ -56,7 +56,8 @@ groups:
       - >
         files.include('site/en/docs/devtools/*') or
         files.include('site/_data/docs/devtools/*') or
-        files.include('site/_data/i18n/docs/devtools.yml')
+        files.include('site/_data/i18n/docs/devtools.yml') or
+        files.include('site/en/blog/*').exclude('site/en/blog/blog.11tydata.js')
```

Comment:
can this include "site/en/blog/*/**.md"? Or do we want it for every file?

Reply:
Good question, I was leaving it as every file for now in case folks (for whatever reason) ever need to link to something like a .zip or a .txt. But yeah, we could probably change them all to *.md as well, I think we'd want to update everything in the file though.

Reply:
Sure, this is fine then.